contracts/scripts/upgrade_aligned_contracts.sh

```diff
@@ -21,13 +21,27 @@ forge_output=$(forge script script/upgrade/AlignedLayerUpgrader.s.sol \
 echo "$forge_output"
 
 # Extract the alignedLayerServiceManagerImplementation value from the output
+aligned_layer_service_manager=$(echo "$forge_output" | awk '/0: address/ {print $3}')
 new_aligned_layer_service_manager_implementation=$(echo "$forge_output" | awk '/1: address/ {print $3}')
 
 # Use the extracted value to replace the alignedLayerServiceManagerImplementation value in alignedlayer_deployment_output.json and save it to a temporary file
-jq --arg new_aligned_layer_service_manager_implementation "$new_aligned_layer_service_manager_implementation" '.addresses.alignedLayerServiceManagerImplementation = $new_aligned_layer_service_manager_implementation' "script/output/holesky/alignedlayer_deployment_output.json" > "script/output/holesky/alignedlayer_deployment_output.temp.json"
+jq --arg new_aligned_layer_service_manager_implementation "$new_aligned_layer_service_manager_implementation" '.addresses.alignedLayerServiceManagerImplementation = $new_aligned_layer_service_manager_implementation' $OUTPUT_PATH > "script/output/holesky/alignedlayer_deployment_output.temp.json"
 
 # Replace the original file with the temporary file
-mv "script/output/holesky/alignedlayer_deployment_output.temp.json" "script/output/holesky/alignedlayer_deployment_output.json"
+mv "script/output/holesky/alignedlayer_deployment_output.temp.json" $OUTPUT_PATH
 
 # Delete the temporary file
 rm -f "script/output/holesky/alignedlayer_deployment_output.temp.json"
+
+data=$(cast calldata "upgrade(address, address)" $aligned_layer_service_manager $new_aligned_layer_service_manager_implementation)
+
+if [ "$MULTISIG" = false ]; then
+  echo "Executing upgrade transaction"
+  proxy_admin=$(jq -r '.addresses.alignedLayerProxyAdmin' $OUTPUT_PATH)
+  cast send $proxy_admin $data \
+    --rpc-url $RPC_URL \
+    --private-key $PRIVATE_KEY
```

Comment:
Note by checking this way omitting the parameter will fall through to the else. Is this the desired behavior?

Reply:
Catched omit case with `-z` at the start of this file